### ISSUE TITLE
Plot collapses on adding multiple labels above/below the plot if title='none' not specified in Plot()

### ISSUE THREAD

**Issue by hkhare42** [bokeh/bokeh#4972]
Attached the images for reference. Without adding labels, one gets the normal plot (Fig 1). With one label added 'above', it works fine (Fig 2). When two labels are added though, whole plot collapses. (Fig 3)

Code:

```
from bokeh.plotting import show, output_file
from bokeh.models import Plot, Label, Circle, ColumnDataSource, DataRange1d, LinearAxis

output_file('gitex.html')

xdr = DataRange1d()
ydr = DataRange1d()

p = Plot(x_range=xdr, y_range=ydr, plot_width=300, plot_height=300, background_fill_color='#F0F0F0')

source = ColumnDataSource(dict(x=[1.0, 2.0, 3.0], y=[1.0, 2.0, 3.0]))

xaxis = LinearAxis()
p.add_layout(xaxis, 'below')

yaxis = LinearAxis()
p.add_layout(yaxis, 'left')

gly = Circle(x='x', y='y', size=10)
p.add_glyph(source, gly)

lab1 = Label(text='Label1', x=0, y=0, x_units='screen', y_units='screen')
p.add_layout(lab1, 'above')

lab2 = Label(text='Label2', x=0, y=0, x_units='screen', y_units='screen')
p.add_layout(lab2, 'above')

show(p)
```

Setting title=None in Plot() works as a quick fix. Mailing list reference for discussion on this with @birdsarah : https://groups.google.com/a/continuum.io/forum/#!topic/bokeh/_dKphJePDwg
Figure1: 
![figure1](https://cloud.githubusercontent.com/assets/10695685/17768111/3ad5115a-6550-11e6-83cb-d77df1fab6c4.png)
Figure2: 
![figure2](https://cloud.githubusercontent.com/assets/10695685/17768112/3b5bc560-6550-11e6-84bc-2008137db59a.png)
Figure3: 
![figure3](https://cloud.githubusercontent.com/assets/10695685/17768114/3e743c82-6550-11e6-80bf-f2ded6a5b8a7.png)

**Comment by birdsarah** [comment]
Can you please include a complete minimal runnable example.

**Comment by hkhare42** [comment]
Updated the initial post with a runnable example.

**Comment by birdsarah** [comment]
Awesome - thanks!

**Comment by birdsarah** [comment]
related to #4561

**Comment by bryevdv** [comment]
Just confirming that this is not fixed by any work up to 0.12.6dev7
